src/main/java/com/kodilla/ecommercee/domain/dto/ProductDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Getter
+@AllArgsConstructor
+public class ProductDto {
+
+    private Long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private int groupId;
+}
```

Comment:
we might need getters as well in that class (I don't know if this is the time when we create them, just suggestion)

Reply:
We can do this now, or with real implementation. I did not use .get methods with example objects . We add this when We will need it